### DISCUSSION TITLE
import bitbucket repo

### DISCUSSION THREAD

**Discussion by fs-ashish** [community/community#54356] [Repositories]
### Select Topic Area

Bug

### Body

when we are importing repository from bitbucket it is showing error. I have attached the screenshot. The error is "there was an error on pushing commits to github".
![Screenshot 2023-05-02 at 12 50 42 PM](https://user-images.githubusercontent.com/129482199/235927036-2d8acdd8-7922-4d7f-a128-a11b628c5ad0.png)

**Comment by herrye991** [comment]
You can import normally at [https://github.com/new/import](https://github.com/new/import). make sure the repository you want to import is public repository and HTTP repository url. for example `https://[username]@bitbucket.org/[username]/[repository].git`. Good Luck 😁

**Comment by klo2k** [comment]
One item to check is "Block command line pushes that expose my email " Email setting:
https://github.com/settings/emails

Unchecking it before running import fixed it for me.
